### PR TITLE
fix: review sonar bugs

### PR DESCRIPTION
**Pull request by Matiasnickolas**
review some bugs reported by Sonar Cloud.

- [x] Add table description
- [x] Delete unused table
- [x] Review uninitialized variable
- [x] Add generic font family
- [x] Change some tables to grid

opencart

### CODE REVIEW

**Review by mvarlic — APPROVED** [review]
LGTM

File: src/upload/admin/view/template/extension/payment/webpay_rest.twig [hunk 1]
```diff
@@ -438,50 +438,54 @@
...
-                                </td>
-                                <td>
+                        <div class="register-container">
+                            <div class="info-column" >
+                                <span class="label label-info info-header" title="{{ label_regs_status }}">
+                                    {{ l_regs_status }}
+                                </span>
+                            </div>
+                            <div class="info-column">
+                                <div class="info-value">
```
Comment (mastudillot) [inline src/upload/admin/view/template/extension/payment/webpay_rest.twig:454]: This change is a refactor, no a fix

File: src/upload/admin/view/template/extension/payment/webpay_rest.twig [hunk 2]
```diff
@@ -211,44 +235,48 @@
...
+                                {{ healthcheck.server_resume.plugin_info.ecommerce }}
+                            </div>
+                        </div>
+                        <div class="info-column" >
+                            <span class="label label-info info-header" title="{{ label_ecommerce_version }}">
+                                {{ l_version }}
+                            </span>
+                        </div>
+                        <div class="info-column">
+                            <div class="info-value">
```
Comment (mastudillot) [inline src/upload/admin/view/template/extension/payment/webpay_rest.twig:268]: Same here, this change is a refactor, no a fix

File: src/upload/admin/view/template/extension/payment/webpay_rest.twig [hunk 3]
```diff
@@ -362,43 +369,68 @@
...
-                            <tr id="row_error_detail" style="display:none">
-                                <td>
-                                    <div title="Detalle del error devuelto por Transbank al fallar init_transaction" class="label label-info">?</div> <strong>Detalle: </strong>
-                                </td>
-                                <td id="row_error_detail_text" class="tbk_table_trans"></td>
-                            </tr>
-                        </tbody>
-                    </table>
+                    <h4 id="response_title" style="display:none">Respuesta de Transbank</h4>
+                    <div class="tbk-response-container" style="display:none" id="div_response_status">
```
Comment (mastudillot) [inline src/upload/admin/view/template/extension/payment/webpay_rest.twig:379]: Same here, this change is a refactor, no a fix

File: src/upload/admin/view/template/extension/payment/webpay_rest.twig [hunk 4]
```diff
@@ -362,43 +369,68 @@
...
-                            </tr>
-                            <tr id="row_error_detail" style="display:none">
-                                <td>
-                                    <div title="Detalle del error devuelto por Transbank al fallar init_transaction" class="label label-info">?</div> <strong>Detalle: </strong>
-                                </td>
-                                <td id="row_error_detail_text" class="tbk_table_trans"></td>
-                            </tr>
-                        </tbody>
-                    </table>
+                    <h4 id="response_title" style="display:none">Respuesta de Transbank</h4>
```
Comment (mastudillot) [inline src/upload/admin/view/template/extension/payment/webpay_rest.twig:378]: This can be inside a class

**Review by mastudillot — COMMENTED** [review]
Please check my comments, and it is better to move this `style="display:none"` to a class

**Pull request closed without merging**